controllers/front/StoresController.php

```diff
@@ -225,6 +225,9 @@ public function getTemplateVarStores()
                 ],
             ];
             $store['image'] = $imageRetriever->getImage(new Store($store['id_store']), $store['id_store']);
+            if (is_array($store['image']['legend'])) {
+                $store['image']['legend'] = $store['image']['legend'][$this->context->language->id];
```

Comment:
are you sure about this ? 
$imageRetriever->getImage returns an array: https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Image/ImageRetriever.php#L178
So you can test is_array($store['image']) but, if we see here: 
```php 
return array(
    'bySize' => $urls,
    'small'  => $small,
    'medium' => $medium,
    'large'  => $large,
    'legend' => isset($object->meta_title) ? $object->meta_title : $object->name,
);
```
legend is alway a string so.. i don't understand this line.